.github/workflows/main.yml

```diff
@@ -65,8 +65,10 @@ jobs:
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
 
-      - name: Install tree
-        run: sudo apt-get install tree
+      - name: Install dependcies
+        run: |
+          sudo apt-get install tree
+          sudo apt-get install bc
```

Comment:
Apparently, `bc` is not installed on the GitHub Action Runners. I think the more proper way would be to use other tools than `bc` to keep the workflow portable, right?

Reply:
Removed `bc` in b96ca21

Reply:
bc should be added to the conda env. But you have found a bash only solution?

Reply:
Yes - instead of piping to bc, I wrapped `zcat {input.fastq1} | wc -l) / 4` in an arithmetic expression `$((...))`. 

There are of course other options if arithmetic expressions are not appropriate: https://www.shell-tips.com/bash/math-arithmetic-calculation/